easypost/services/base_service.py

```diff
@@ -26,11 +26,11 @@ def _snakecase_name(self, class_name) -> str:
 
     def _class_url(self, class_name) -> str:
```

Comment:
This method continues to be the source of my nightmares 😉

Reply:
If this is your nightmares, you'll do just fine 😉 It's only worked fine for a decade now in multiple libs.